lib/northbound_cli.c

```diff
@@ -550,6 +550,13 @@ void nb_cli_show_config_prepare(struct nb_config *config, bool with_defaults)
 				       LYD_VALIDATE_NO_STATE, NULL);
 }
 
+static int lyd_node_sort(struct lyd_node **dnode1, struct lyd_node **dnode2)
```

Comment:
Could we call this `_compare` or `_cmp` instead, just to align with other sort patterns?

Reply:
Done.